perma_web/replay/templates/iframe.html

```diff
@@ -21,31 +33,59 @@
       font-family: "Roboto", Helvetica, Arial, "Lucida Grande", sans-serif;
       font-size: 16px;
       line-height: 22.85px;
-      padding: 14px 24px ;
+      padding: 14px 24px;
       text-align: center;
-      white-space: nowrap
+      white-space: nowrap;
+      text-decoration: none;
     }
-    #download-interstitial:hover, #download-interstitial:focus {
+    
+    .button:hover, .button:focus {
       background-color: #0092FF;
     }
+
     .button-wrapper {
       margin: auto;
       text-align: center;
     }
-    replay-web-page {
-      display: block;
-    }
-    .hidden {
-      visibility: hidden;
-      height: 10px;
-      min-width: 10px;
-      width: 10px
+
+    /* Fallback screen when hotlinked or missing args */
+    #no-playback, #no-playback * {
+      box-sizing: border-box;
     }
-    .welcome {
+
+    #no-playback {
       height: 100%;
       display: flex;
       align-items: center;
       justify-content: center;
+      font-family: "Roboto", Helvetica, Arial, "Lucida Grande", sans-serif;
+      font-weight: 300;
+      color: #333333;
+      padding: 1rem;
+    }
+
+    #no-playback div {
+      text-align: center;
+    }
+
+    #no-playback p {
+      font-size: clamp(20px, 3vmax, 36px);
```

Comment:
I just started using `clamp()` and I love it.

Reply:
Ooh, I have not yet learned about `clamp()`.